lib/opsgenie_alert_tagging.rb

```diff
@@ -11,6 +11,7 @@ def initialize(date)
   end
 
   def call
+    raise "Need to supply date" if date.nil?
```

Comment:
Why not do this on `initialize` instead? That's the point where a date needs to be provided.